src/infra/bus.js

```diff
@@ -14,7 +17,24 @@ class Bus {
   /** @type BusInterface['command'] */
   command({ service: serviceName, method }, payload) {
     const service = this.#services.get(serviceName);
-    return service[method](payload);
+    if (!service) {
+      return Promise.resolve([
+        { expected: false, message: 'Service not found' },
+        null,
+      ]);
+    }
+
+    const handler = service[method];
+    if (!handler) {
+      return Promise.resolve([
+        { expected: false, message: 'Method not found' },
+        null,
+      ]);
+    }
+
+    const operationId = randomUUID();
+    const wrappedMeta = { ...payload.meta, operationId };
+    return handler({ ...payload, meta: wrappedMeta });
```

Comment:
If you want to use [error, result] contract - better to wrap outside, whenever a command called